### PR TITLE
Implement Trello integration

### PR DESCRIPTION
**Pull request by angelsanzn**
Includes two notification strategies:

```
- card-wise: for each notification, creates a new item
in a specified checklist of a specified card.

- list-wise: for each notification, creates a new card
in a specified list.
```

### CODE REVIEW

**Comment by ergl** [comment]
Nice! Taking a look at it now

File: spec/trello_integration_spec.py [hunk 1]
```diff
@@ -0,0 +1,254 @@
+import json
+
+from expects import expect, raise_error, match, be_a, equal, have_key
+from doublex import Spy
+from doublex_expects import have_been_called_with
+
+from pysellus.stock_integrations import trello
+from pysellus.interfaces import AbstractIntegration
```
Comment (ergl) [inline spec/trello_integration_spec.py]: seems like `AbstractIntegration` is not used here?

Comment (angelsanzn) [inline spec/trello_integration_spec.py]: Thanks, fixed.

**Comment by ergl** [comment]
`CardWiseNotifier` and `ListWiseNotifier` look almost identical, do you think you could abstract them together?

Also, I see that there's only `on_next` in here — perhaps an `on_error` implementation could be useful as well?

**Comment by ergl** [comment]
Also, for this integration, how would an config entry look like?

**Comment by angelsanzn** [comment]
The config entry would just declare constructor parameters:

``` yaml
    'my-trello-alias':
        trello:
            key: ...
            token: ...
            card: ...
            checklist: ...
            mode: list
            list: ...
```

As long as you declare the required parameters (and only those) for the selected (or default) mode, it works.

**Comment by ergl** [comment]
Just getting back to this now — as I said, everything looks good, but I think that the two strategies can be abstracted a bit, right now they look too similar.

What do you think?

**Comment by angelsanzn** [comment]
Yes, I refactored in order to tighten the responsibilities of each class and pushed between yesterday and just a few moments ago. What do you think of it as it stands now? I believe it's ready.

(I also implemented `on_error` and `on_completed`.)

**Comment by ergl** [comment]
Yeah, this looks much better, good job!

**Comment by angelsanzn** [comment]
Thanks!

**Pull request merged**